librz/include/rz_util/rz_file.h

```diff
@@ -45,6 +45,8 @@ RZ_API char *rz_file_path_unix_to_local(const char *path);
 RZ_API char *rz_file_binsh(void);
 RZ_API ut8 *rz_inflatew(const ut8 *src, int srcLen, int *srcConsumed, int *dstLen, int wbits);
 RZ_API ut8 *rz_inflate(const ut8 *src, int srcLen, int *srcConsumed, int *dstLen);
+RZ_API ut8 *rz_deflatew(const ut8 *src, int srcLen, int *srcConsumed, int *dstLen, int wbits);
```

Comment:
When you add those `RZ_*` markers in the function definition, please add them in header files as well. Would be awesome to have them for the `rz_inflate*()` functions as well, btw.

Reply:
Oh, I am sorry. My bad. Thank you. :)